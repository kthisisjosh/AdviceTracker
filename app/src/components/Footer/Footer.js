import React from "react"
import { Typography } from "@material-ui/core"

export const Footer = () => {
    return (
        <footer style={{ backgroundColor: "#FFFFFF", margin: "0", width: "100vw", height: "190px", textAlign: "center" }}>
            <Typography variant="h5" justify="center" style={{ color: "black", paddingTop: "75px" }}>
                Advice Tracker
            </Typography>
            <Typography variant="body2" style={{ margin: "auto", marginTop: "35px", lineHeight: "24px" }}>
                © Advice Tracker 2020
            </Typography>
            <Typography variant="body2" style={{ margin: "auto", lineHeight: "24px" }}>
                Made with{" "}
                <span role="img" aria-label="Heart">
                    ❤️
                </span>{" "}
                by{" "}
                <a style={{ textDecoration: "none", color: "inherit" }} href="https://www.joshbautista.com">
                    Joshua Bautista
                </a>
            </Typography>
        </footer>
    )
}

export default Footer
