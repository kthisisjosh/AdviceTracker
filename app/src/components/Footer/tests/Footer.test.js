import React from "react"
import renderer from "react-test-renderer"
import Footer from "../Footer"

describe("<Footer />", () => {
    it("should render and match the snapshot", () => {
        const renderedComponent = renderer.create(<Footer />).toJSON()

        expect(renderedComponent).toMatchSnapshot()
    })


})
