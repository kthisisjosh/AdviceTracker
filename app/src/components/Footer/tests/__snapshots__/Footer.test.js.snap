// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render and match the snapshot 1`] = `
<footer
  style={
    Object {
      "boxSizing": "border-box",
      "display": "inline-block",
      "height": "auto",
      "margin": "0",
      "right": "0",
      "textAlign": "center",
      "width": "100vw",
    }
  }
>
  <div
    style={
      Object {
        "marginTop": "5vh",
      }
    }
  >
    <a
      href="/"
    >
      <img
        alt="logo"
        src="https://advicetracker.life/AdviceLogo.png"
      />
    </a>
  </div>
  <hr
    style={
      Object {
        "width": "auto",
      }
    }
  />
  <p
    className="MuiTypography-root MuiTypography-body2"
    style={
      Object {
        "lineHeight": "24px",
        "margin": "auto",
        "marginTop": "35px",
      }
    }
  >
    © Advice Tracker 2020
  </p>
  <p
    className="MuiTypography-root MuiTypography-body2"
    style={
      Object {
        "lineHeight": "24px",
        "margin": "auto",
      }
    }
  >
    Made with
     
    <span
      aria-label="Heart"
      role="img"
    >
      ❤️
    </span>
     
    by
     
    <a
      href="https://www.joshbautista.com"
      style={
        Object {
          "color": "inherit",
        }
      }
    >
      Joshua Bautista
    </a>
  </p>
</footer>
`;
